### PR TITLE
updated alfred (3.0_652)

### PR DESCRIPTION
**Pull request by rajiv**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/alfred.rb [hunk 1]
```diff
@@ -20,11 +20,16 @@
             login_item: 'Alfred 2'
 
   zap delete: [
+                '~/Library/Application Support/Alfred 3',
```
Comment (vitorgalvao) [inline Casks/alfred.rb]: Remove the references to Alfred 2.

Comment (rajiv) [inline Casks/alfred.rb]: i was following the existing example in the the cask as it has references to alfred 1 ("alfred" without the version number). if you are removing, zapping this cask, shouldn't it remove files from all the previous versions?

Comment (vitorgalvao) [inline Casks/alfred.rb]: That’s still something we’re working on. We’ll save a snapshot of the cask you actually installed so we can remove that.

File: Casks/alfred.rb [hunk 2]
```diff
@@ -20,11 +20,16 @@
             login_item: 'Alfred 2'
```
Comment (adidalal) [inline Casks/alfred.rb]: Should this also be changed? Don't use Alfred, so not sure

Comment (vitorgalvao) [inline Casks/alfred.rb]: Yes. And the `quit:` as well.

File: Casks/alfred.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
   auto_updates true
   accessibility_access true
 
-  app 'Alfred 2.app'
+  app 'Alfred 3.app'
```
Comment (vitorgalvao) [inline Casks/alfred.rb]: `app "Alfred #{version.major}.app"`

**Pull request merged**